src/utils/database.ts

```
@@ -27,7 +27,7 @@ export const initializeDatabase = async (
   try {
     await db.query(transaction.statements.map((statement) => statement.statement));
   } catch (_error) {
-    console.log('ronin_schema already exists');
```

We need to wrap the query in a try-catch block to handle potential failures if the `ronin_schema` already exists. This prevents unnecessary console spam.

---

In a subsequent iteration, let's parse the error, otherwise we're silently ignoring all errors, even if the error is something completely different than a "table already exists" error.